src/logger/MeetingSessionPOSTLogger.ts

```diff
@@ -30,11 +30,18 @@ export default class MeetingSessionPOSTLogger {
       });
   }
 
-  debug(debugFunction: () => string): void {
+  debug(debugFunction: string | (() => string)): void {
     if (LogLevel.DEBUG < this.level) {
       return;
     }
-    this.log(LogLevel.DEBUG, debugFunction());
+
+    if (typeof debugFunction === 'string') {
+      this.log(LogLevel.DEBUG, debugFunction);
+    } else if (debugFunction) {
+      this.log(LogLevel.DEBUG, debugFunction());
+    } else {
+      this.log(LogLevel.DEBUG, '' + debugFunction);
```

Comment:
So is this just a ''+null/undefined?

Reply:
Yes; if you call `.debug(undefined)`, it's equivalent to calling `.debug('undefined')`.